lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +26,27 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
```

Comment:
It's still going to blow up since the new code expects the new kwargs to be there.